TowerDefense/MouseHandler.cs

```diff
@@ -31,10 +31,8 @@ public enum SelectionContext
         PlacingCheese,
         TowerSelected
     }
-    class MouseHandler
+    class MouseHandler : GameObject
```

Comment:
Making the mouse handler a game-object seemed weird.... but if you select a portal it needs to animate :)